### DISCUSSION TITLE
Output the telemetry data using the serial port, but I don't know the data protocol

### DISCUSSION THREAD

**Discussion by wsnfly** [EdgeTX/edgetx#2890] [General]
I want to use the aircraft's telemetry data (GPS and altitude) for AAT via the serial output on the bottom of the T16 remote. Of course, I made it output data, but I don't know what protocol data it is, at least it is not MAVLINK protocol.

**Comment by raphaelcoeffic** [comment]
When using "Telemetry Mirror" as the Serial Function for that particular serial port, EdgeTx will mirror the exact payload as received from the radio module. As an example, if your module is talking CRSF, the telemetry mirror feed will be CRSF.
